src/ui/marker.js

```diff
@@ -31,6 +34,9 @@ type Options = {
  * @param {PointLike} [options.offset] The offset in pixels as a {@link PointLike} object to apply relative to the element's center. Negatives indicate left and up.
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
  * @param {boolean} [options.draggable=false] A boolean indicating whether or not a marker is able to be dragged to a new position on the map.
+ * @param {number} [options.rotatation=0] The rotation angle of the marker (in degrees), relative to it's respective {@link Marker#rotationAlignment} setting.
+ * @param {string} [options.pitchAlignment='auto'] The pitch alignment for the marker, allowing it to stay flat on the map or to stay vertical relative to the viewport.
+ * @param {string} [options.rotationAlignment='auto'] The rotation alignment for the marker, allowing for rotations relative to either the viewport or the map.
```

Comment:
I think this needs to make it clear that the options are `map` and `viewport`, how about this:

    `map` aligns the `Marker`'s rotation relative to the map, `viewport` aligns the `Marker`'s rotation relative to viewport.

Reply:
Used this for both `pitchAlignment` and `rotationAlignment`